lms/djangoapps/mobile_api/models.py

```diff
@@ -2,8 +2,9 @@
 ConfigurationModel for the mobile_api djangoapp.
 """
 from django.db import models
-from mobile_api import utils
+
 from config_models.models import ConfigurationModel
+from mobile_api import utils
```

Comment:
You can do `from . import utils`, but if you leave out the `from .`, you introduce ambiguity, and break forward compatibility with python 3.